src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/EnclaveTest.java

```diff
@@ -0,0 +1,630 @@
+/*
```

Comment:
I am confused about packages. AETest.java is in com.microsoft.sqlserver.jdbc package, but EnclaveTest.java is in com.microsoft.sqlserver.jdbc.AlwaysEncrypted package.